sdk/src/main/java/co/omise/android/ui/PaymentCreatorActivity.kt

```diff
@@ -405,7 +503,7 @@ private class PaymentCreatorNavigationImpl(
 interface PaymentCreatorRequester<T : Model> {
     val amount: Long
     val currency: String
-    val capability: Capability
+    var capability: Capability
```

Comment:
Is this intentionally making capability mutable?

Reply:
it should not be mutable, was part of a different approach that i was trying. Will restore it.